packages/npm-dependencies-validation/src/utils/packageJsonUtil.ts

```diff
@@ -17,17 +18,18 @@ const pnpmManagerFiles = [
 ];
 const monorepoProps = ["workspaces"];
 
-async function readJsonFile(jsonFilePath: string): Promise<any> {
+async function readJsonFile(jsonFilePath: string): Promise<PackageJson> {
   try {
     const packageJsonContent = await readFile(jsonFilePath, "utf-8");
-    const content: { name: string } = JSON.parse(packageJsonContent);
+    const content: PackageJson = JSON.parse(packageJsonContent);
     return content;
   } catch (error) {
-    return {};
+    // TODO: should we return null/undefined ???
```

Comment:
should we log something in the catch block?

Reply:
I do not think we need log here, the question is what should we to return in this case ?